src/main/java/seedu/address/model/AddressBookHistory.java

```diff
@@ -17,10 +17,20 @@ public AddressBookHistory(ReadOnlyAddressBook addressBook) {
         addressBooks.add(new AddressBook(addressBook));
     }
 
+    public ArrayList<AddressBook> getAddressBooks() {
```

Comment:
I don't think a getter violates a feature freeze, but still have concerns about this. Should be fine though.

Reply:
@medajet  Thanks for bringing this up. Since this command (`getAddressBooks()`) is not used anywhere else, I think it does not fix any bugs/add any features to TAPA. I think it exists solely to improve tests. However, I will work on your suggestion above, which definitely changes the behaviour of TAPA.